.depalignrc.json

```diff
@@ -26,6 +26,9 @@
     ],
     "js-yaml": [
       "^3.13.1"
+    ],
+    "yargs": [
+      "^4.8.1"
```

Comment:
hadron-build is on 4 and I don't feel like I can deal with it as a drive by of adding yargs usage to compass-e2e-tests tbh